tests/cross/src/test/scala/tests/pc/CompletionOverrideSuite.scala

```diff
@@ -211,7 +247,17 @@ class CompletionOverrideSuite extends BaseCompletionSuite {
       |  }
       |}
     """.stripMargin,
-    filter = _.contains("iterat")
+    filter = _.contains("iterat"),
+    compat = Map(
+      "3" -> // TODO: should s/Any/Iterator[A]/
+        """
+          |object Main {
+          |  new scala.Iterable[Unknown] {
+          |    def iterator: Any = ${0:???}
```

Comment:
Let's fix this one before merging

Reply:
Now it's better https://github.com/scalameta/metals/pull/3897/commits/43253abd7cc4f72aa35ad94850d38d1ef48f233b but I'm not sure how can we make it `Iterable[A]` instead of `Iterable[Any]` 🤔 
Can we work on that in another issue?

Reply:
https://github.com/scalameta/metals/issues/3926